File: scratch/inspect_db.js
async function run() {
    const DB_URL = "https://gestion-team-e-default-rtdb.firebaseio.com";
    try {
        const resp = await fetch(`${DB_URL}/state/statuses.json`);
        const data = await resp.json();
        console.log("Firebase statuses keys count:", Object.keys(data || {}).length);
        console.log("Sample of statuses:", Object.entries(data || {}).slice(0, 5));
    } catch(e) {
        console.error("Error fetching:", e);
    }
}
run();

File: scratch/inspect_db.py
import urllib.request
import json

url = "https://gestion-team-e-default-rtdb.firebaseio.com/state.json"
try:
    with urllib.request.urlopen(url) as response:
        state = json.loads(response.read().decode())
        rps = state.get("rps")
        print("Type of rps in Firebase:", type(rps))
        if isinstance(rps, dict):
            print("Keys:", list(rps.keys())[:10])
        elif isinstance(rps, list):
            print("Length:", len(rps))
            print("First 5 items:", rps[:5])
except Exception as e:
    print("Error:", e)
